src/plugins/LADTable/components/LADTableConfiguration.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+<div class="c-inspect-properties">
+    <template v-if="isEditing">
+        <div class="c-inspect-properties__header">
+            Table Column Visibility
+        </div>
+        <ul class="c-inspect-properties__section">
+            <li
+                v-for="(title, key) in headers"
+                :key="key"
+                class="c-inspect-properties__row"
+            >
+                <div
+                    class="c-inspect-properties__label"
+                    title="Show or hide column"
+                >
+                    <label :for="key + 'ColumnControl'">{{ title }}</label>
+                </div>
+                <div class="c-inspect-properties__value">
+                    <input
+                        :id="key + 'ColumnControl'"
+                        type="checkbox"
+                        :checked="configuration.hiddenColumns[key] !== true"
+                        @change="toggleColumn(key)"
+                    >
+                </div>
+            </li>
+        </ul>
+    </template>
+</div>
+</template>
+
+<script>
+
+export default {
+    inject: ['openmct', 'ladTableConfiguration'],
+    data() {
+        return {
+            headers: {
+                timestamp: 'timestamp',
```

Comment:
We should capitalize these I think to match the other property headers
<img width="248" alt="image" src="https://user-images.githubusercontent.com/9853862/223044402-48ad0d9a-8515-4dd3-b6b1-e568d137abf6.png">

Reply:
Good call!